opencti-platform/opencti-graphql/tests/02-integration/01-database/middleware-test.js

```diff
@@ -751,9 +751,9 @@ describe('Relations distribution', () => {
     expect(distribution[0].entity.representative).toEqual({ main: 'Restricted', secondary: 'Restricted' });
     expect(distribution[1].entity.representative).toEqual({ main: 'Restricted', secondary: 'Restricted' });
     expect(distribution[2].entity.representative).toEqual({ main: 'Restricted', secondary: 'Restricted' });
-    expect(distribution[0].entity.name).toBeUndefined();
-    expect(distribution[1].entity.name).toBeUndefined();
-    expect(distribution[2].entity.name).toBeUndefined();
+    expect(distribution[0].entity.name).toEqual('Restricted');
+    expect(distribution[1].entity.name).toEqual('Restricted');
+    expect(distribution[2].entity.name).toEqual('Restricted');
```

Comment:
you could also check the presence of other fields like standard_id ? I think this was the one faulty in the original issue.

Or alter another test to query more things and check it does not crash because of missing fields.